content/themes/casper/assets/css/screen.css

```diff
@@ -1742,28 +2026,71 @@ body:not(.post-template) .post-title {
 
 
 /* ==========================================================================
-   12. Media Queries - Smaller than 500px
+   12. Media Queries - Smaller than 768px
    ========================================================================== */
 
-@media only screen and (max-width: 500px) {
+@media only screen and (max-width: 768px) {
 
-    .main-header {
-        margin-bottom: 15px;
-        height: 40vh;
+    .content {
+        flex-direction: column-reverse;
```

Comment:
make it cross-browser compatible:
flex-direction, -moz-flex-direction, -webkit-flex-direction